tsconfig.json

```diff
@@ -16,7 +15,8 @@
     "strictNullChecks": true,
     "noFallthroughCasesInSwitch": true,
     "noUnusedParameters": true,
-    "noUnusedLocals": true
+    "noUnusedLocals": true,
+    "declaration": true
```

Comment:
Nit: could you put this bellow line 4? I like to put all of the file output options together 😊